README.md

```diff
@@ -230,6 +230,36 @@ sudo systemctl enable et.service
 sudo systemctl start et.service
 ```
 
+### Raspbian (Stretch):
+## (_tested on_: Raspberry Pi 3B+):
+
++ Install dependencies
+
+	sudo apt install python3-protobuf protobuf-compiler
+
++ Install from source
+```
+mkdir ~/tmp
+cd ~/tmp
```

Comment:
nit: Maybe we do not need to create `tmp` folder here?
Reason being `git clone` will create a project folder by default, as result we can omit this step.